.travis.yml

```diff
@@ -10,7 +10,9 @@ script:
 after_success:
   "curl $AUTODEPLOY_WEBHOOK -H \"token: $AUTODEPLOY_TOKEN\""
 
+cache: pip
+
 notifications:
   email:
     on_success: never
-    on_failure: never
\ No newline at end of file
+    on_failure: never
```

Comment:
This file is out of scope of the PR, please remove it.

Reply:
I don't even know where that line came from, probably my auto-travis-lint?